environment_setup/install_clion.sh

```diff
@@ -42,10 +42,10 @@ echo "================================================================"
 echo "Installing Bazel Plugin"
 echo "================================================================"
 
-clion_plugin_dir="${HOME}/.CLion${clion_version_year}.${clion_version_major}/config/plugins"
-bazel_plugin_version_year="2019"
-bazel_plugin_version_major="12"
-bazel_plugin_version_minor="16"
+clion_plugin_dir="${HOME}/.CLion${clion_version_year}.{${clion_version_major}}/config/plugins"
```

Comment:
@garethellis0 didn't notice this until I ran locally. Why add the extra `{}`? Not it shows up in the filename which seems odd

`~/.CLion2019.{3}`

Reply:
It showed up in the filename for me for some reason.